src/theme/foundations/typography.ts

```diff
@@ -267,12 +245,6 @@ const typography: Typography = {
       subtitle1: fontWeightValues["regular"],
       subtitle2: fontWeightValues["regular"],
     },
-    text: {
-      default: fontWeightValues["light"],
-      caption: fontWeightValues["light"],
-      tag: fontWeightValues["regular"],
-      mini: fontWeightValues["regular"],
-    },
```

Comment:
`"text.default"` is still used in `global.ts`.

Reply:
These values were updated in `3.6.1`, so they are back in with the new values.

Reply:
Ok, I see what's happening. In `global.ts`, `text.default` should be replaced with `body.default`. These tokens were deprecated and can be removed.